addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3837,7 +3837,7 @@ export class OdooEditor extends EventTarget {
     }
 
     getCurrentCollaborativeSelection() {
-        const selection = this._latestComputedSelection || this._computeHistorySelection();
```

Comment:
I think this fix also applies in 15.0.

Reply:
Thanks for the heads-up! I didn't check 15.0 cause it works well there. But after I compared these two versions and found it's actually because the `this.getCurrentCollaborativeSelection()` is now executed after `this._computeHistorySelection();` 

The change is made here: https://github.com/odoo/odoo/commit/28a35a6b53493abd03e42cc30d164262c86b6de5 and I figured that this commit is more about the `appliedCustomSelection` related code. So I propose a new fix that separates the `appliedCustomSelection` and `collaboration` (tested and works for the collaboration), this will keep both fixes works

```js
const appliedCustomSelection = selection.rangeCount && selection.getRangeAt(0) ? this._handleSelectionInTable() : false;

if (!appliedCustomSelection) {
    this._updateToolbar(!selection.isCollapsed && this.isSelectionInEditable(selection));
}

......

if (
    selection.rangeCount &&
    selection.getRangeAt(0) &&
    this.options.onCollaborativeSelectionChange
) {
    this.options.onCollaborativeSelectionChange(this.getCurrentCollaborativeSelection());
}
```
the history of the change:
https://github.com/odoo/odoo/commit/28a35a6b53493abd03e42cc30d164262c86b6de5 (move the position of collaboration code)
https://github.com/odoo/odoo/commit/3d9be78fbc6f8471c9bb24368b3299b89d914fa1 (add appliedCustomSelection to collaboration)